logical/framework/testdata/operations.json

```diff
@@ -0,0 +1,86 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "HashiCorp Vault API",
+    "description": "HTTP API that gives you full access to Vault. All API routes are prefixed with `/v1/`.",
```

Comment:
Can we describe an `api_version`? So rather than hardcode prefix here in the description we can indicate that it will be prefixed by the current `api_version`? Or is this standard fields and that won't fit in?

Reply:
Something like `v1` in OpenAPI has an official home but is now part of the "server" object (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#server-object) which doesn't really fit what we're doing for our OpenAPI output, and I definitely don't want to prefix each of the actual paths with `/v1`.

Here's one idea. 1. Define "/v1" in code. 2. Make the description string use that value 3. Add `x-api-version` to the `info` path as well, and reference that in the description. That way readers know about it clearly, but clients can pick it off programatically.

Reply:
This would only be at the top level (not per path). But, it's easily added later so I say let's leave as-is and think on it for now.